addons/spreadsheet_dashboard/static/src/bundle/dashboard_action/dashboard_action.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <templates>
     <div t-name="spreadsheet_dashboard.DashboardAction" class="o_action o_spreadsheet_dashboard_action o_field_highlight">
-        <ControlPanel display="controlPanelDisplay">
+        <ControlPanel display="controlPanelDisplay" t-if="!state.activeDashboard?.isSample">
```

Comment:
Do not hide entirely the ControlPanel, but the gf (and the Share button, wait for FRGI input)